openeo_udp/sentinel1_stats/generate.py

```diff
@@ -0,0 +1,74 @@
+import json
+from pathlib import Path
+
+import openeo
+from openeo.api.process import Parameter
+from openeo.rest.udp import build_process_dict
+from openeo.processes import array_create, array_concat
+
+# TODO #15 where to put reusable helpers? e.g. load description from README.md, dummy openeo connection, properly write to JSON file, ...
+
+
+def generate() -> dict:
+    # TODO: use/inject dummy connection instead of concrete one? (See cdse_marketplace_services)
+    connection = openeo.connect("openeofed.dataspace.copernicus.eu")
+
+    # define parameters
+    spatial_extent = Parameter.bounding_box(
```

Comment:
use spatial_extent
```suggestion
    spatial_extent = Parameter.spatial_extent( default={"west": 8.82, "south": 44.40, "east": 8.92, "north": 44.45})
```
(hope that suggestion works...)